addons/l10n_in_ewaybill_stock/views/l10n_in_ewaybill_views.xml

```diff
@@ -129,7 +120,6 @@
                             <field name="move_ids" mode="tree,kanban" force_save="1" readonly="state != 'pending'">
                                 <tree editable="bottom" create="0" delete="0">
                                     <field name="company_currency_id" column_invisible="1"/>
```

Comment:
@smetl Do you know a way to get rid of this `company_currency_id `? The issue on removing it is that we currency symbol is no longer visible on the tree view

Reply:
@hamo-odoo I think we need it here anyways as it is the currency_field of ewaybill_priceunit